src/mono/wasm/runtime-test.js

```diff
@@ -119,6 +119,7 @@ setenv = {};
 runtime_args = [];
 enable_gc = true;
 enable_zoneinfo = false;
+working_dir='/';
```

Comment:
what is the default working_dir generall?

Reply:
I tried printing cwd for other tests, and it was `/`.

Reply:
I mean, we weren't setting it at all. And `FS.cwd` said `/`